src/kuzzle.js

```diff
@@ -1,9 +1,9 @@
 var
   uuid = require('uuid'),
-  KuzzleDataCollection = require('./kuzzleDataCollection'),
-  KuzzleSecurity = require('./security/kuzzleSecurity'),
-  KuzzleMemoryStorage = require('./kuzzleMemoryStorage'),
-  KuzzleUser = require('./security/kuzzleUser'),
+  Collection = require('./kuzzleCollection'),
```

Comment:
Ok, so I propose to follow the same naming conventions as other SDK: let's make id TitleCase.

* Kuzzle.js
* Collection.js
* CollectionMapping.js
* Document.js,
* MemoryStorage.js
* Room.js
* SearchResult.js
* SubscribeResult